woocommerce/woocommerce-paypal-payments
# remove currencies from country-currency matrix in card fields module (2665)

### Description



We need to remove the currency from the country-currency matrix for card fields module.

### Steps to Test




1. Go to WooCommerce -> Settings -> General and choose a supported country;
2. Observe the card fields in checkout;
3. Go to WooCommerce -> Settings -> General and choose an unsupported country;
4. Observe the absence of the card fields in checkout.
